ldclient/flag.py

```diff
@@ -10,13 +10,18 @@
 import sys
 
 from ldclient import operators
+from ldclient.util import stringify_attrs
 from ldclient.versioned_data_kind import FEATURES, SEGMENTS
 
 __LONG_SCALE__ = float(0xFFFFFFFFFFFFFFF)
 
 __BUILTINS__ = ["key", "ip", "country", "email",
                 "firstName", "lastName", "avatar", "name", "anonymous"]
 
+__USER_ATTRS_TO_STRINGIFY_FOR_EVALUATION__ = [ "key", "secondary" ]
+# Currently we are not stringifying the rest of the built-in attributes prior to evaluation, only for events.
+# This is because it could affect evaluation results for existing users (ch35206).
```

Comment:
Should we expose a clubhouse ticket number?

Reply:
I don't see why not. It's not sensitive information and it may be helpful to us later.

Reply:
Fair enough, just checking.